layouts/partials/head/sticky-header.html

```diff
@@ -0,0 +1,13 @@
+{{ if .Site.Params.sticky_header }}
+<style>
+    :target {
+        scroll-margin-top: 80px;
+    }
+
+    @media (min-width: 1000px) {
+        :target {
+            scroll-margin-top: 100px;
```

Comment:
Ideally take into account the height of the browser to see if the header is, in fact, sticky.

Reply:
Updated media query